eng/pipelines/templates/jobs/archetype-sdk-tests-jobs.yml

```diff
@@ -0,0 +1,111 @@
+parameters:
+- name: Platforms
+  type: object
+  default: []
+- name: PreSteps
+  type: object
+  default: []
+- name: PostSteps
+  type: object
+  default: []
+- name: EnvVars
+  type: object
+  default: {}
+- name: MaxParallel
+  type: number
+  default: 0
+- name: BuildInParallel
+  type: boolean
+  default: true
+- name: TimeoutInMinutes
+  type: number
+  default: 60
+- name: Location
+  type: string
+  default: ''
+- name: SubscriptionConfiguration
+  type: string
+  default: $(sub-config-azure-cloud-test-resources)
+- name: ServiceDirectory
+  type: string
+  default: not-specified
+
+jobs:
+  - ${{ each platform in parameters.Platforms }}:
+    - job:
+      displayName: ${{ platform.DisplayName }}
+      variables:
+        - template: ../variables/globals.yml
+        - name: OSVmImage
+          value: ${{ platform.OSVmImage }}
+
+      timeoutInMinutes: ${{ parameters.TimeoutInMinutes }}
+
+      pool:
+        vmImage: $(OSVmImage)
+      ${{ if platform.Container }}:
```

Comment:
Is this alignment correct? I would have expected it to be one more level indented. Although I'm not sure.